.gitignore

```diff
@@ -18,8 +18,8 @@ _build
 .customizer
 
 bower_components
-bundle
-build/*
+./bundle
```

Comment:
Why different variants for paths? With / without dot, with / wuthout leading slash and so on?

Reply:
See https://www.atlassian.com/git/tutorials/saving-changes/gitignore

> - `/debug.log`
>   Prepending a slash matches files only in the repository root. 
> - `logs`
>    If you don't append a slash, the pattern will match both files and the contents of directories with that name. In the example matches on the left, both directories and files named logs are ignored 
> - `logs/`
>    Appending a slash indicates the pattern is a directory. The entire contents of any directory in the repository matching that name – including all of its files and subdirectories – will be ignored

But the `./` prefix is the same as `/`